src/tokens/ods/color.yaml

```diff
@@ -77,6 +77,8 @@ color:
   background:
     default:
       value: rgb(255, 255, 255)
+    secondary:
+      value: rgb(236, 236, 236)
```

Comment:
I'm still confused. You wanted to set the current `background-highlight` color value here for the `background-secondary` and then come up with a new color value for `background-highlight` which better visualizes a highlighted state.
Or in other words: We want to keep the color value for the OcDrop background, but the variable name that was used there is semantically questionable. `background-secondary` would be a better naming for the OcDrop background, but the actual color of the OcDrop is supposed to stay as it is.

Reply:
discussed in a call that the code it's good like it is code wise 👍